components/screens/BlogScreen/BlogStyles.js

```diff
@@ -0,0 +1,41 @@
+import { StyleSheet } from 'react-native';
+
+export const Styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    marginHorizontal: 16,
+  },
+  title: {
+    textAlign: 'center',
+    marginVertical: 8,
+    fontSize: 22,
+    // backgroundColor: '#434345',
```

Comment:
I think it'd be nice to have this cleaned up, unless you are using this later and need it in the code.

Reply:
Done